Essentials/src/messages.properties

```diff
@@ -608,8 +608,10 @@ versionOutputUnsupported=\u00a7d{0} \u00a76version: \u00a7d{1}
 versionOutputUnsupportedPlugins=\u00a76You are running \u00a7dunsupported plugins\u00a76!
 versionMismatch=\u00a74Version mismatch\! Please update {0} to the same version.
 versionMismatchAll=\u00a74Version mismatch\! Please update all Essentials jars to the same version.
-voiceSilenced=\u00a76Your voice has been silenced\!
-voiceSilencedReason=\u00a76Your voice has been silenced\! Reason: \u00a7c{0}
+voiceSilenced=\u00a76Your voice has been silenced for {0}\!
```

Comment:
I'm pretty sure this line wasn't meant to be changed

Reply:
Intentional